05/app.js

```diff
@@ -9,12 +9,46 @@ const stats = {
 
 /* tutaj umieść swój kod */
 
+const paragraphElements = document.querySelectorAll('p')
+const anchorElements = document.querySelectorAll('a')
+
+const paragraphArray = [...paragraphElements]
+
+paragraphArray.forEach(function(item){
+    const paragraphID = item.dataset.id
+    stats.paragraphs[paragraphID] = 0
+})
```

Comment:
Trochę sobie ułatwiłeś tym rozwiązaniem, ale doceniam, że problem rozwiązałeś.
Jednak z perspektywy optymalizacji nie jest to najlepsze rozwiążanie.
Gdybyśmy mieli 100 czy 1000 elementów, lepiej pokazywać tylko te kliknięte.